build.sh

```diff
@@ -32,6 +32,15 @@ if ! ls "${ZIPNAME}" 1> /dev/null 2>&1; then
     exit 1
 fi
 
+if [ -z "${BASE_DOCKER_IMAGE}" ]; then
```

Comment:
This should be renamed to ${RESOLVE_BASE_IMAGE}.

1.  It helps to avoid possible collisions.
2.  It adheres to the convention of prepending RESOLVE_ to all configuration settings.
3.  As this repo supports both docker and podman containers, it is agnostic to which one.
4. It is easier to associate the to the internally used BASE_IAMGE.